Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs

```diff
@@ -290,7 +290,8 @@ protected virtual void UpdateCursorTransform()
             GameObject newTargetedObject = focusDetails.Object;
 
             // Get the forward vector looking back along the pointing ray.
-            Vector3 lookForward = -Pointer.Ray.direction;
```

Comment:
Because this field was moved a few lines below the comment got left behind.  Should we also update this with a `<summary>` tag?

Reply:
Doesn't strike me as necessary but I'll follow your lead.